purldb-toolkit/pyproject.toml

```diff
@@ -1,15 +1,130 @@
+[build-system]
+requires = [ "flot>=0.7.0" ]
+build-backend = "flot.buildapi"
+
 [project]
 name = "purldb-toolkit"
-version = "0.0.1"
+version = "0.1.0"
+description = "A toolkit and library to use the PurlDB and its API"
+readme = "README.rst"
+license = { text = "Apache-2.0" }
+requires-python = ">=3.7"
 
-[build-system]
-requires = ["setuptools == 68.2.2", "wheel", "setuptools_scm[toml] >= 6"]
-build-backend = "setuptools.build_meta"
+authors = [
+    { name = "nexB. Inc. and others", email = "info@aboutcode.org" },
+]
+
+keywords = [
+    "purl",
+    "Package-URL",
+    "Git",
+    "metadata",
+    "package",
+    "dependencies",
+    "open source",
+    "package",
+    "sca",
+    "scan",
+]
+
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Software Development",
+    "Topic :: Utilities",
+]
+
+dependencies = [
+    "click >= 6.7, !=7.0",
+    "fetchcode",
+    "packageurl_python >= 0.9.0",
```

Comment:
Does this `packageurl_python` need to be changed to `>= 0.13.4` ?  I'd previously had to update the `setup.cfg` (which I think was replaced by this `pyproject.toml`) and `requirements.txt` for `packageurl-python` from 0.11.2 to 0.13.4 so that several of my pkg:gem vs. pkg:rubygems tests would pass.